CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ Note: For changes to the API, see https://shopify.dev/changelog?filter=api
 
 ## Unreleased
 
-N/A
+- [#1002](https://github.com/Shopify/shopify-api-ruby/pull/1002) Add new method to construct the host app URL
```

Comment:
Should we mention that this enables apps to redirect back to the appropriate Shopify surface, so it's safer than `shop`?